gradle.properties

```diff
@@ -2,7 +2,7 @@ ideaVersion=IC-2019.3
 # build number ranges
 # https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html
 sinceIdeaBuild=193
-projectVersion=0.3.1-SNAPSHOT
+projectVersion=0.4.0
```

Comment:
Why the SNAPSHOT disappeared ?

Reply:
want to release 0.4.0, for this sake need a stable version which will get bumped to 0.4.0-SNAPSHOT once released and tagged. Missing something?

Reply:
So you want to release 0.4.0 ? Then you also need a modification in plugin.xml to document changes, no ?

Reply:
ouch, forgot, again, thx for reminding me. doing.

Reply:
@jeffmaury plz re-review